aten/src/ATen/core/Backtrace.h

```diff
@@ -1,28 +1,3 @@
-#pragma once
+#include "c10/util/Backtrace.h"
```

Comment:
The reason for leaving such forwarding headers behind is mostly backwards compatibility because legacy third party code we don't have control over might include them. I don't think that applies to `ATen/core` since the folder is so new. We can probably just delete this header and refactor the files including it.

Reply:
Thanks for pointing it out - yes, this one is mainly because there are too many indirect dependencies caused (mainly stl headers) so I put it here intending to remove it in an immediate follow up PR.